back/package.json

```diff
@@ -11,16 +11,21 @@
   "author": "",
   "license": "ISC",
   "dependencies": {
+    "bcrypt": "^5.1.1",
     "cors": "^2.8.5",
     "dotenv": "^16.3.1",
     "express": "^4.18.2",
+    "jsonwebtoken": "^9.0.2",
     "mysql2": "^3.6.5",
```

Comment:
remove it if possible

Reply:
It's to automatically parse json body

Reply:
Like to generate jwt, and I need it to import and not require

Reply:
I miss click i want to click on the line under : mysql2